src/api/java/com/minecolonies/api/crafting/IRecipeStorageFactory.java

```diff
@@ -68,6 +68,11 @@ default RecipeStorage getNewInstance(@NotNull final IFactoryController factoryCo
             throw new IllegalArgumentException("Eighth parameter is supposed to be a List<ItemStack> or Null!");
         }
 
+        if (context.length > MIN_PARAMS_IRECIPESTORAGE + 5 && context[8] != null && !(context[8] instanceof ResourceLocation))
+        {
+            throw new IllegalArgumentException("Eighth parameter is supposed to be a List<ItemStack> or Null!");
```

Comment:
either the message is off here, or sth else is wrong

Reply:
Dangit. The message is off. Fixing.